homeassistant/components/lifx/coordinator.py

```diff
@@ -381,20 +355,126 @@ def async_get_active_effect(self) -> int:
         """Return the enum value of the currently active firmware effect."""
         return self.active_effect.value
 
-    async def async_set_hev_cycle_state(self, enable: bool, duration: int = 0) -> None:
-        """Start or stop an HEV cycle on a LIFX Clean bulb."""
-        if lifx_features(self.device)["hev"]:
-            await async_execute_lifx(
-                partial(self.device.set_hev_cycle, enable=enable, duration=duration)
-            )
+
+class LIFXSensorUpdateCoordinator(DataUpdateCoordinator):
+    """DataUpdateCoordinator to gather data for a specific lifx device."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        parent: LIFXUpdateCoordinator,
+        title: str,
+    ) -> None:
+        """Initialize DataUpdateCoordinator."""
+        self.parent: LIFXUpdateCoordinator = parent
+        self.lock = asyncio.Lock()
```

Comment:
Shouldnt this share the parent lock?

Reply:
Strictly speaking, this coordinator doesn't actually need a lock at all as it doesn't modify any visible characteristics, i.e. it doesn't visibly change the state of a light except when applying a theme and that hands off the application to the `aiolifx_themes` package.